x-pack/plugins/reporting/server/core.ts

```diff
@@ -127,12 +126,8 @@ export class ReportingCore {
     return this.exportTypesRegistry;
   }
 
-  public async getEsqueue() {
-    return (await this.getPluginStartDeps()).esqueue;
-  }
-
-  public async getEnqueueJob() {
-    return (await this.getPluginStartDeps()).enqueueJob;
```

Comment:
We didn't need this: the function can just be imported from `routes/generation.ts` where it is needed